[2023-10-04T12:05:09.669+0000] {processor.py:157} INFO - Started process (PID=542) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:09.677+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:05:09.686+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:09.682+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:09.874+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:10.105+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:10.104+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:05:10.173+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:10.173+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:05:10.248+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.609 seconds
[2023-10-04T12:05:40.570+0000] {processor.py:157} INFO - Started process (PID=547) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:40.571+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:05:40.572+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:40.572+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:40.600+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:05:40.636+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:40.636+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:05:40.662+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:05:40.662+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:05:40.695+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.130 seconds
[2023-10-04T12:06:10.803+0000] {processor.py:157} INFO - Started process (PID=552) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:10.804+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:06:10.806+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:10.805+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:10.837+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:10.883+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:10.883+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:06:10.903+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:10.902+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:06:10.943+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.143 seconds
[2023-10-04T12:06:41.139+0000] {processor.py:157} INFO - Started process (PID=557) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:41.141+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:06:41.143+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:41.142+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:41.188+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:06:41.228+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:41.227+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:06:41.254+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:06:41.254+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:06:41.301+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.166 seconds
[2023-10-04T12:07:11.376+0000] {processor.py:157} INFO - Started process (PID=562) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:11.377+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:07:11.378+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:11.378+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:11.406+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:11.440+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:11.440+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:07:11.454+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:11.453+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:07:11.474+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.102 seconds
[2023-10-04T12:07:42.216+0000] {processor.py:157} INFO - Started process (PID=567) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:42.224+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:07:42.231+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:42.229+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:42.359+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:07:42.500+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:42.500+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:07:42.545+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:07:42.545+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:07:42.585+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.394 seconds
[2023-10-04T12:08:12.855+0000] {processor.py:157} INFO - Started process (PID=572) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:12.857+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:08:12.859+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:12.858+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:12.887+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:12.921+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:12.920+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:08:12.943+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:12.943+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:08:12.983+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.131 seconds
[2023-10-04T12:08:43.250+0000] {processor.py:157} INFO - Started process (PID=577) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:43.255+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:08:43.261+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:43.259+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:43.380+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:08:43.528+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:43.528+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:08:43.624+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:08:43.624+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:08:43.709+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.487 seconds
[2023-10-04T12:09:14.048+0000] {processor.py:157} INFO - Started process (PID=582) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:14.050+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:09:14.052+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:14.052+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:14.108+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:14.167+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:14.167+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:09:14.186+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:14.186+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:09:14.210+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.204 seconds
[2023-10-04T12:09:44.550+0000] {processor.py:157} INFO - Started process (PID=587) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:44.558+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:09:44.566+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:44.563+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:44.735+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:09:44.886+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:44.886+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:09:44.934+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:09:44.933+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:09:44.978+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.468 seconds
[2023-10-04T12:10:15.267+0000] {processor.py:157} INFO - Started process (PID=592) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:15.269+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:10:15.271+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:15.270+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:15.303+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:15.348+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:15.348+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:10:15.399+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:15.399+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:10:15.433+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.172 seconds
[2023-10-04T12:10:45.680+0000] {processor.py:157} INFO - Started process (PID=597) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:45.685+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:10:45.687+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:45.687+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:45.734+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:10:45.775+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:45.775+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:10:45.790+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:10:45.789+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:10:45.810+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.140 seconds
[2023-10-04T12:11:15.926+0000] {processor.py:157} INFO - Started process (PID=602) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:15.927+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:11:15.928+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:15.928+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:15.954+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:15.988+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:15.988+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:11:16.001+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:16.001+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:11:16.023+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.101 seconds
[2023-10-04T12:11:47.358+0000] {processor.py:157} INFO - Started process (PID=607) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:47.381+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:11:47.403+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:47.392+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:47.724+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:11:47.998+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:47.998+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:11:48.067+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:11:48.067+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:11:48.119+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.881 seconds
[2023-10-04T12:12:18.416+0000] {processor.py:157} INFO - Started process (PID=612) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:18.418+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:12:18.419+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:18.418+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:18.476+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:18.611+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:18.610+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:12:18.651+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:18.651+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:12:18.688+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.294 seconds
[2023-10-04T12:12:48.966+0000] {processor.py:157} INFO - Started process (PID=617) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:48.969+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:12:48.971+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:48.970+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:49.032+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:12:49.111+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:49.110+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:12:49.137+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:12:49.137+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:12:49.162+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.209 seconds
[2023-10-04T12:13:19.246+0000] {processor.py:157} INFO - Started process (PID=622) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:19.247+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:13:19.250+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:19.248+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:19.289+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:19.368+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:19.368+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:13:19.397+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:19.396+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:13:19.432+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.190 seconds
[2023-10-04T12:13:49.633+0000] {processor.py:157} INFO - Started process (PID=627) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:49.634+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:13:49.635+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:49.635+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:49.665+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:13:49.701+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:49.701+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:13:49.739+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:13:49.739+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:13:49.763+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.133 seconds
[2023-10-04T12:14:20.341+0000] {processor.py:157} INFO - Started process (PID=632) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:20.349+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:14:20.355+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:20.353+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:20.489+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:20.625+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:20.625+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:14:20.671+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:20.671+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:14:20.710+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.395 seconds
[2023-10-04T12:14:50.974+0000] {processor.py:157} INFO - Started process (PID=637) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:50.975+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:14:50.976+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:50.976+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:51.005+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:14:51.042+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:51.042+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:14:51.075+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:14:51.075+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:14:51.109+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.139 seconds
[2023-10-04T12:15:21.206+0000] {processor.py:157} INFO - Started process (PID=642) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:21.207+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:15:21.208+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:21.208+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:21.245+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:21.282+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:21.281+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:15:21.296+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:21.296+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:15:21.322+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.120 seconds
[2023-10-04T12:15:51.422+0000] {processor.py:157} INFO - Started process (PID=647) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:51.424+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:15:51.425+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:51.425+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:51.450+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:15:51.483+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:51.483+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:15:51.494+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:15:51.494+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:15:51.517+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.098 seconds
[2023-10-04T12:16:22.124+0000] {processor.py:157} INFO - Started process (PID=652) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:22.140+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:16:22.152+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:22.147+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:22.649+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:23.607+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:23.601+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:16:23.800+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:23.800+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:16:23.969+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 1.888 seconds
[2023-10-04T12:16:54.615+0000] {processor.py:157} INFO - Started process (PID=657) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:54.618+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:16:54.622+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:54.621+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:54.724+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:16:54.824+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:54.824+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:16:54.856+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:16:54.855+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:16:54.891+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.299 seconds
[2023-10-04T12:17:25.122+0000] {processor.py:157} INFO - Started process (PID=662) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:25.124+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:17:25.128+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:25.127+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:25.188+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:25.247+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:25.247+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:17:25.271+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:25.271+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:17:25.306+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.194 seconds
[2023-10-04T12:17:55.588+0000] {processor.py:157} INFO - Started process (PID=667) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:55.589+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:17:55.591+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:55.590+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:55.624+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:17:55.661+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:55.661+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:17:55.676+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:17:55.676+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:17:55.699+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.117 seconds
[2023-10-04T12:18:25.907+0000] {processor.py:157} INFO - Started process (PID=672) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:25.908+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:18:25.910+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:25.909+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:25.938+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:25.969+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:25.969+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:18:25.984+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:25.984+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:18:26.006+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.102 seconds
[2023-10-04T12:18:56.636+0000] {processor.py:157} INFO - Started process (PID=677) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:56.643+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:18:56.649+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:56.646+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:56.762+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:18:56.925+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:56.924+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:18:56.973+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:18:56.973+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:18:57.012+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.398 seconds
[2023-10-04T12:19:27.285+0000] {processor.py:157} INFO - Started process (PID=682) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:27.289+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:19:27.291+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:27.290+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:27.348+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:27.436+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:27.436+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:19:27.458+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:27.458+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:19:27.489+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.214 seconds
[2023-10-04T12:19:57.697+0000] {processor.py:157} INFO - Started process (PID=687) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:57.698+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:19:57.699+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:57.699+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:57.724+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:19:57.756+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:57.756+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:19:57.769+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:19:57.769+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:19:57.789+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.096 seconds
[2023-10-04T12:20:27.903+0000] {processor.py:157} INFO - Started process (PID=692) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:27.904+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:20:27.905+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:27.905+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:27.941+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:27.980+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:27.979+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:20:27.993+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:27.992+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:20:28.014+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.115 seconds
[2023-10-04T12:20:58.553+0000] {processor.py:157} INFO - Started process (PID=697) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:58.563+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:20:58.571+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:58.567+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:58.764+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:20:58.949+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:58.949+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:20:59.014+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:20:59.014+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:20:59.074+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.554 seconds
[2023-10-04T12:21:29.447+0000] {processor.py:157} INFO - Started process (PID=702) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:29.450+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:21:29.453+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:21:29.452+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:29.532+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:29.649+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:21:29.649+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:21:29.687+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:21:29.687+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:21:29.721+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.286 seconds
[2023-10-04T12:21:59.920+0000] {processor.py:157} INFO - Started process (PID=707) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:59.921+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:21:59.922+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:21:59.922+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:59.947+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:21:59.985+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:21:59.985+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:22:00.008+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:22:00.007+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:22:00.048+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.131 seconds
[2023-10-04T12:22:30.136+0000] {processor.py:157} INFO - Started process (PID=712) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:22:30.137+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:22:30.139+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:22:30.138+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:22:30.172+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:22:30.212+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:22:30.211+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:22:30.227+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:22:30.227+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:22:30.253+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.121 seconds
[2023-10-04T12:23:01.467+0000] {processor.py:157} INFO - Started process (PID=717) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:23:01.475+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:23:01.504+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:23:01.493+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:23:02.043+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:23:02.678+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:23:02.677+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:23:02.843+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:23:02.843+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:23:02.951+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 1.578 seconds
[2023-10-04T12:34:59.667+0000] {processor.py:157} INFO - Started process (PID=172) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:34:59.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:34:59.729+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:34:59.720+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:35:00.429+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:35:02.027+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:02.026+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-10-04T12:35:02.041+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:02.040+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-10-04T12:35:02.054+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:02.054+0000] {manager.py:499} INFO - Created Permission View: %s
[2023-10-04T12:35:02.055+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:02.055+0000] {security.py:708} INFO - Not syncing DAG-level permissions for DAG 'DAG:etl_dag' as access control is unset.
[2023-10-04T12:35:02.057+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:02.056+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:35:03.972+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:03.972+0000] {dag.py:2937} INFO - Creating ORM DAG for etl_dag
[2023-10-04T12:35:03.974+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:03.974+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:35:04.000+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 4.430 seconds
[2023-10-04T12:35:34.341+0000] {processor.py:157} INFO - Started process (PID=177) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:35:34.345+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:35:34.350+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:34.348+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:35:34.438+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:35:34.616+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:34.616+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:35:34.704+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:35:34.703+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:35:34.780+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.459 seconds
[2023-10-04T12:36:05.043+0000] {processor.py:157} INFO - Started process (PID=182) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:05.047+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:36:05.051+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:05.050+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:05.114+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:05.229+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:05.228+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:36:05.262+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:05.261+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:36:05.298+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.270 seconds
[2023-10-04T12:36:35.653+0000] {processor.py:157} INFO - Started process (PID=187) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:35.657+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:36:35.663+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:35.661+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:35.853+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:36:36.110+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:36.109+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:36:36.160+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:36:36.160+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:36:38.062+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 2.430 seconds
[2023-10-04T12:37:08.322+0000] {processor.py:157} INFO - Started process (PID=192) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:08.325+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:37:08.327+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:08.326+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:08.358+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:08.410+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:08.409+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:37:08.873+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:08.873+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:37:08.903+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.588 seconds
[2023-10-04T12:37:39.178+0000] {processor.py:157} INFO - Started process (PID=197) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:39.184+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:37:39.185+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:39.184+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:39.215+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:37:39.422+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:39.422+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:37:39.436+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:37:39.435+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:37:39.459+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.298 seconds
[2023-10-04T12:38:09.677+0000] {processor.py:157} INFO - Started process (PID=202) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:09.678+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:38:09.679+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:09.679+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:09.819+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:09.847+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:09.847+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:38:09.858+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:09.858+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:38:09.881+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.208 seconds
[2023-10-04T12:38:40.883+0000] {processor.py:157} INFO - Started process (PID=207) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:40.901+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:38:40.918+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:40.912+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:41.217+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:38:41.763+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:41.762+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:38:41.977+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:38:41.977+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:38:42.203+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 1.383 seconds
[2023-10-04T12:39:12.555+0000] {processor.py:157} INFO - Started process (PID=212) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:12.559+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:39:12.562+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:12.560+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:12.638+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:12.764+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:12.764+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:39:12.805+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:12.805+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:39:12.850+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.312 seconds
[2023-10-04T12:39:43.082+0000] {processor.py:157} INFO - Started process (PID=217) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:43.084+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:39:43.085+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:43.084+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:43.114+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:39:43.196+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:43.196+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:39:43.228+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:39:43.228+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:39:43.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.179 seconds
[2023-10-04T12:40:13.334+0000] {processor.py:157} INFO - Started process (PID=222) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:13.336+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:40:13.338+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:13.338+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:13.367+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:13.412+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:13.411+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:40:13.428+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:13.428+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:40:13.467+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.137 seconds
[2023-10-04T12:40:43.689+0000] {processor.py:157} INFO - Started process (PID=227) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:43.692+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:40:43.693+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:43.693+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:43.734+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:40:43.783+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:43.783+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:40:43.804+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:40:43.804+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:40:43.827+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.149 seconds
[2023-10-04T12:41:14.431+0000] {processor.py:157} INFO - Started process (PID=232) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:14.436+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:41:14.442+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:14.440+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:14.546+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:14.700+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:14.700+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:41:14.745+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:14.745+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:41:14.788+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.377 seconds
[2023-10-04T12:41:45.047+0000] {processor.py:157} INFO - Started process (PID=237) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:45.050+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:41:45.053+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:45.052+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:45.096+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:41:45.156+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:45.156+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:41:45.175+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:41:45.175+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:41:45.209+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.170 seconds
[2023-10-04T12:42:15.428+0000] {processor.py:157} INFO - Started process (PID=242) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:15.429+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:42:15.430+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:15.430+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:15.456+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:15.489+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:15.489+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:42:15.503+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:15.502+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:42:15.523+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.100 seconds
[2023-10-04T12:42:45.679+0000] {processor.py:157} INFO - Started process (PID=247) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:45.680+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:42:45.681+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:45.681+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:45.707+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:42:45.739+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:45.739+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:42:45.753+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:42:45.753+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:42:45.780+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.104 seconds
[2023-10-04T12:43:16.286+0000] {processor.py:157} INFO - Started process (PID=252) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:16.291+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:43:16.297+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:16.294+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:16.446+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:16.598+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:16.598+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:43:16.654+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:16.653+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:43:16.710+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.448 seconds
[2023-10-04T12:43:46.941+0000] {processor.py:157} INFO - Started process (PID=257) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:46.942+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:43:46.944+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:46.943+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:46.978+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:43:47.015+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:47.015+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:43:47.028+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:43:47.028+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:43:47.053+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.117 seconds
[2023-10-04T12:44:17.154+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:17.157+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:44:17.159+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:17.159+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:17.199+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:17.243+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:17.242+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:44:17.264+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:17.263+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:44:17.321+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.172 seconds
[2023-10-04T12:44:47.506+0000] {processor.py:157} INFO - Started process (PID=267) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:47.508+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:44:47.509+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:47.509+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:47.542+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:44:47.582+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:47.582+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:44:47.596+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:44:47.595+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:44:47.616+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.117 seconds
[2023-10-04T12:45:19.163+0000] {processor.py:157} INFO - Started process (PID=272) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:19.201+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:45:19.229+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:19.216+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:19.809+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:20.321+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:20.320+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:45:20.460+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:20.459+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:45:20.546+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 1.562 seconds
[2023-10-04T12:45:50.883+0000] {processor.py:157} INFO - Started process (PID=277) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:50.885+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:45:50.886+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:50.886+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:50.921+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:45:50.956+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:50.956+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:45:50.971+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:45:50.970+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:45:50.991+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.114 seconds
[2023-10-04T12:46:21.109+0000] {processor.py:157} INFO - Started process (PID=282) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:21.111+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:46:21.112+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:21.112+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:21.149+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:21.191+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:21.190+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:46:21.208+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:21.207+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:46:21.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.138 seconds
[2023-10-04T12:46:51.456+0000] {processor.py:157} INFO - Started process (PID=287) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:51.457+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:46:51.458+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:51.457+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:51.484+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:46:51.515+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:51.515+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:46:51.527+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:46:51.527+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:46:51.547+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.094 seconds
[2023-10-04T12:47:21.668+0000] {processor.py:157} INFO - Started process (PID=292) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:21.670+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:47:21.671+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:21.671+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:21.713+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:21.812+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:21.812+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:47:21.831+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:21.831+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:47:21.858+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.194 seconds
[2023-10-04T12:47:52.410+0000] {processor.py:157} INFO - Started process (PID=297) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:52.414+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:47:52.419+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:52.416+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:52.534+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:47:52.693+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:52.692+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:47:52.743+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:47:52.743+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:47:52.781+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.399 seconds
[2023-10-04T12:48:23.040+0000] {processor.py:157} INFO - Started process (PID=302) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:23.044+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:48:23.048+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:48:23.047+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:23.097+0000] {processor.py:839} INFO - DAG(s) dict_keys(['etl_dag']) retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:23.185+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:48:23.185+0000] {dag.py:2915} INFO - Sync 1 DAGs
[2023-10-04T12:48:23.206+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:48:23.206+0000] {dag.py:3696} INFO - Setting next_dagrun for etl_dag to None, run_after=None
[2023-10-04T12:48:23.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.203 seconds
[2023-10-04T12:48:53.530+0000] {processor.py:157} INFO - Started process (PID=307) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:53.532+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:48:53.533+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:48:53.533+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:53.587+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:48:53.566+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:48:53.589+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:48:53.632+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.107 seconds
[2023-10-04T12:49:23.748+0000] {processor.py:157} INFO - Started process (PID=312) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:23.750+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:49:23.751+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:23.751+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:23.769+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:23.765+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:49:23.770+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:23.798+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.053 seconds
[2023-10-04T12:49:28.875+0000] {processor.py:157} INFO - Started process (PID=317) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:28.876+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:49:28.877+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:28.877+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:28.910+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:28.905+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:49:28.912+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:28.936+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.064 seconds
[2023-10-04T12:49:59.589+0000] {processor.py:157} INFO - Started process (PID=322) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:59.595+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:49:59.602+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:59.599+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:59.677+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:49:59.657+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:49:59.681+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:49:59.794+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.229 seconds
[2023-10-04T12:50:30.144+0000] {processor.py:157} INFO - Started process (PID=327) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:50:30.147+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:50:30.148+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:50:30.148+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:50:30.174+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:50:30.168+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:50:30.175+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:50:30.200+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.060 seconds
[2023-10-04T12:51:00.337+0000] {processor.py:157} INFO - Started process (PID=332) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:00.338+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:51:00.340+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:51:00.339+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:00.362+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:51:00.356+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:51:00.363+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:00.389+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.055 seconds
[2023-10-04T12:51:30.632+0000] {processor.py:157} INFO - Started process (PID=337) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:30.634+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:51:30.635+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:51:30.635+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:30.656+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:51:30.651+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:51:30.658+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:51:30.692+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.064 seconds
[2023-10-04T12:52:01.265+0000] {processor.py:157} INFO - Started process (PID=342) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:01.274+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:52:01.281+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:52:01.278+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:01.400+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:52:01.372+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:52:01.408+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:01.539+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.303 seconds
[2023-10-04T12:52:31.814+0000] {processor.py:157} INFO - Started process (PID=347) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:31.815+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:52:31.817+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:52:31.816+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:31.839+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:52:31.834+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:52:31.840+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:52:31.867+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.057 seconds
[2023-10-04T12:53:02.098+0000] {processor.py:157} INFO - Started process (PID=352) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:02.099+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:53:02.100+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:53:02.100+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:02.120+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:53:02.116+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:53:02.122+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:02.151+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.055 seconds
[2023-10-04T12:53:32.374+0000] {processor.py:157} INFO - Started process (PID=357) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:32.380+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:53:32.383+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:53:32.382+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:32.416+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:53:32.410+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:53:32.419+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:53:32.515+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.147 seconds
[2023-10-04T12:54:03.063+0000] {processor.py:157} INFO - Started process (PID=362) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:03.072+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:54:03.078+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:54:03.076+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:03.209+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:54:03.177+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:54:03.219+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:03.413+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.387 seconds
[2023-10-04T12:54:34.104+0000] {processor.py:157} INFO - Started process (PID=367) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:34.109+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:54:34.114+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:54:34.111+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:34.163+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:54:34.150+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:54:34.165+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:54:34.274+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.201 seconds
[2023-10-04T12:55:04.512+0000] {processor.py:157} INFO - Started process (PID=372) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:04.514+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:55:04.515+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:55:04.515+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:04.536+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:55:04.531+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:55:04.537+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:04.570+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.063 seconds
[2023-10-04T12:55:34.783+0000] {processor.py:157} INFO - Started process (PID=377) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:34.785+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:55:34.786+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:55:34.786+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:34.805+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:55:34.801+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:55:34.806+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:55:34.835+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.055 seconds
[2023-10-04T12:56:05.072+0000] {processor.py:157} INFO - Started process (PID=382) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:05.073+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:56:05.074+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:56:05.074+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:05.092+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:56:05.088+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:56:05.094+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:05.115+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.045 seconds
[2023-10-04T12:56:35.678+0000] {processor.py:157} INFO - Started process (PID=387) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:35.683+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:56:35.689+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:56:35.687+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:35.737+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:56:35.727+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:56:35.739+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:56:35.842+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.189 seconds
[2023-10-04T12:57:06.117+0000] {processor.py:157} INFO - Started process (PID=392) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:06.119+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:57:06.121+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:57:06.121+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:06.149+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:57:06.143+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:57:06.150+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:06.187+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.078 seconds
[2023-10-04T12:57:36.385+0000] {processor.py:157} INFO - Started process (PID=397) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:36.387+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:57:36.388+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:57:36.388+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:36.412+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:57:36.408+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:57:36.413+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:57:36.453+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.073 seconds
[2023-10-04T12:58:06.639+0000] {processor.py:157} INFO - Started process (PID=402) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:06.641+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:58:06.642+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:58:06.642+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:06.660+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:58:06.656+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:58:06.661+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:06.689+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.053 seconds
[2023-10-04T12:58:37.228+0000] {processor.py:157} INFO - Started process (PID=407) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:37.237+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:58:37.244+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:58:37.241+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:37.345+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:58:37.325+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:58:37.351+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:58:37.457+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.255 seconds
[2023-10-04T12:59:07.692+0000] {processor.py:157} INFO - Started process (PID=412) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:07.695+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:59:07.696+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:59:07.695+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:07.719+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:59:07.712+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:59:07.720+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:07.759+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.075 seconds
[2023-10-04T12:59:37.953+0000] {processor.py:157} INFO - Started process (PID=417) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:37.954+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T12:59:37.955+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:59:37.955+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:37.973+0000] {logging_mixin.py:151} INFO - [2023-10-04T12:59:37.969+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T12:59:37.974+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T12:59:38.006+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.056 seconds
[2023-10-04T13:00:08.353+0000] {processor.py:157} INFO - Started process (PID=422) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:08.354+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:00:08.356+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:00:08.355+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:08.383+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:00:08.376+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:00:08.385+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:08.419+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.075 seconds
[2023-10-04T13:00:39.426+0000] {processor.py:157} INFO - Started process (PID=427) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:39.449+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:00:39.488+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:00:39.480+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:39.684+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:00:39.648+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:00:39.692+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:00:39.995+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.687 seconds
[2023-10-04T13:01:10.253+0000] {processor.py:157} INFO - Started process (PID=432) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:10.254+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:01:10.255+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:01:10.255+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:10.277+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:01:10.272+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:01:10.279+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:10.324+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.075 seconds
[2023-10-04T13:01:40.551+0000] {processor.py:157} INFO - Started process (PID=437) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:40.552+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:01:40.554+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:01:40.553+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:40.573+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:01:40.569+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:01:40.574+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:01:40.604+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.056 seconds
[2023-10-04T13:02:10.775+0000] {processor.py:157} INFO - Started process (PID=442) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:10.776+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:02:10.777+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:02:10.777+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:10.799+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:02:10.795+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:02:10.801+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:10.835+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.063 seconds
[2023-10-04T13:02:41.701+0000] {processor.py:157} INFO - Started process (PID=447) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:41.708+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:02:41.721+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:02:41.714+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:41.863+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:02:41.830+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:02:41.871+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:02:42.123+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.472 seconds
[2023-10-04T13:03:12.590+0000] {processor.py:157} INFO - Started process (PID=452) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:12.595+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:03:12.600+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:03:12.598+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:12.638+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:03:12.629+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:03:12.640+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:12.730+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.163 seconds
[2023-10-04T13:03:42.834+0000] {processor.py:157} INFO - Started process (PID=457) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:42.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:03:42.840+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:03:42.838+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:42.868+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:03:42.862+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:03:42.869+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:03:42.926+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.105 seconds
[2023-10-04T13:04:13.197+0000] {processor.py:157} INFO - Started process (PID=462) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:13.199+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:04:13.200+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:04:13.200+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:13.218+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:04:13.214+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:04:13.220+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:13.242+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.049 seconds
[2023-10-04T13:04:43.451+0000] {processor.py:157} INFO - Started process (PID=467) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:43.453+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:04:43.454+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:04:43.453+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:43.473+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:04:43.469+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:04:43.474+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:04:43.498+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.050 seconds
[2023-10-04T13:05:14.295+0000] {processor.py:157} INFO - Started process (PID=472) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:14.300+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:05:14.305+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:05:14.303+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:14.343+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:05:14.334+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:05:14.345+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:14.410+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.136 seconds
[2023-10-04T13:05:44.521+0000] {processor.py:157} INFO - Started process (PID=477) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:44.523+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:05:44.524+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:05:44.523+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:44.544+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:05:44.540+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:05:44.546+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:05:44.576+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.061 seconds
[2023-10-04T13:06:14.789+0000] {processor.py:157} INFO - Started process (PID=482) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:14.790+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:06:14.791+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:06:14.791+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:14.815+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:06:14.811+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:06:14.817+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:14.889+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.103 seconds
[2023-10-04T13:06:45.099+0000] {processor.py:157} INFO - Started process (PID=487) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:45.104+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:06:45.109+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:06:45.106+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:45.146+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:06:45.136+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:06:45.148+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:06:45.230+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.162 seconds
[2023-10-04T13:07:15.782+0000] {processor.py:157} INFO - Started process (PID=492) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:15.789+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:07:15.796+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:07:15.793+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:15.898+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:07:15.878+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:07:15.905+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:16.004+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.247 seconds
[2023-10-04T13:07:46.303+0000] {processor.py:157} INFO - Started process (PID=497) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:46.305+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:07:46.307+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:07:46.307+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:46.334+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:07:46.329+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:07:46.335+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:07:46.375+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.077 seconds
[2023-10-04T13:08:16.566+0000] {processor.py:157} INFO - Started process (PID=502) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:16.568+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:08:16.570+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:08:16.570+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:16.591+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:08:16.587+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:08:16.593+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:16.622+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.059 seconds
[2023-10-04T13:08:46.864+0000] {processor.py:157} INFO - Started process (PID=507) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:46.865+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:08:46.866+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:08:46.866+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:46.886+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:08:46.882+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:08:46.888+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:08:46.913+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.052 seconds
[2023-10-04T13:09:17.317+0000] {processor.py:157} INFO - Started process (PID=512) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:17.327+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:09:17.338+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:09:17.331+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:17.407+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:09:17.391+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:09:17.410+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:17.573+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.291 seconds
[2023-10-04T13:09:47.925+0000] {processor.py:157} INFO - Started process (PID=517) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:47.926+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:09:47.927+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:09:47.927+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:47.950+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:09:47.945+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:09:47.952+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:09:47.988+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.068 seconds
[2023-10-04T13:10:18.187+0000] {processor.py:157} INFO - Started process (PID=522) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:18.188+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:10:18.189+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:10:18.189+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:18.208+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:10:18.204+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:10:18.210+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:18.238+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.055 seconds
[2023-10-04T13:10:48.416+0000] {processor.py:157} INFO - Started process (PID=527) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:48.417+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:10:48.418+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:10:48.418+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:48.437+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:10:48.433+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:10:48.438+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:10:48.464+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.050 seconds
[2023-10-04T13:11:18.735+0000] {processor.py:157} INFO - Started process (PID=532) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:18.741+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:11:18.744+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:11:18.743+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:18.778+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:11:18.769+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:11:18.780+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:19.014+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.297 seconds
[2023-10-04T13:11:49.425+0000] {processor.py:157} INFO - Started process (PID=537) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:49.435+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:11:49.442+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:11:49.439+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:49.521+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:11:49.502+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:11:49.524+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:11:49.710+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.324 seconds
[2023-10-04T13:12:19.979+0000] {processor.py:157} INFO - Started process (PID=542) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:19.984+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:12:19.990+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:12:19.988+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:20.023+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:12:20.017+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:12:20.025+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:20.089+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.116 seconds
[2023-10-04T13:12:50.224+0000] {processor.py:157} INFO - Started process (PID=547) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:50.225+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:12:50.226+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:12:50.226+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:50.247+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:12:50.243+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:12:50.249+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:12:50.290+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.069 seconds
[2023-10-04T13:13:20.438+0000] {processor.py:157} INFO - Started process (PID=552) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:20.439+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:13:20.440+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:13:20.440+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:20.465+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:13:20.456+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:13:20.467+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:20.501+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.066 seconds
[2023-10-04T13:13:51.047+0000] {processor.py:157} INFO - Started process (PID=557) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:51.056+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:13:51.063+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:13:51.060+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:51.164+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:13:51.145+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:13:51.171+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:13:51.273+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.253 seconds
[2023-10-04T13:14:21.527+0000] {processor.py:157} INFO - Started process (PID=562) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:21.528+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:14:21.529+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:14:21.529+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:21.549+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:14:21.544+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:14:21.550+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:21.582+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.060 seconds
[2023-10-04T13:14:51.723+0000] {processor.py:157} INFO - Started process (PID=567) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:51.724+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:14:51.725+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:14:51.725+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:51.745+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:14:51.740+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:14:51.746+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:14:51.773+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.054 seconds
[2023-10-04T13:15:22.084+0000] {processor.py:157} INFO - Started process (PID=572) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:22.085+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:15:22.087+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:15:22.087+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:22.111+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:15:22.106+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:15:22.113+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:22.144+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.063 seconds
[2023-10-04T13:15:52.869+0000] {processor.py:157} INFO - Started process (PID=577) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:52.877+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:15:52.895+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:15:52.891+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:53.056+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:15:53.011+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:15:53.066+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:15:53.370+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.562 seconds
[2023-10-04T13:16:23.778+0000] {processor.py:157} INFO - Started process (PID=582) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:23.781+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:16:23.782+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:16:23.782+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:23.804+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:16:23.800+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:16:23.805+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:23.858+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.086 seconds
[2023-10-04T13:16:54.054+0000] {processor.py:157} INFO - Started process (PID=587) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:54.055+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:16:54.056+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:16:54.056+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:54.074+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:16:54.070+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:16:54.076+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:16:54.104+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.052 seconds
[2023-10-04T13:17:24.287+0000] {processor.py:157} INFO - Started process (PID=592) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:24.288+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:17:24.289+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:17:24.289+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:24.308+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:17:24.304+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:17:24.309+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:24.331+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.048 seconds
[2023-10-04T13:17:54.586+0000] {processor.py:157} INFO - Started process (PID=597) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:54.587+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:17:54.588+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:17:54.588+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:54.608+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:17:54.604+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:17:54.610+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:17:54.634+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.053 seconds
[2023-10-04T13:18:11.467+0000] {processor.py:157} INFO - Started process (PID=602) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:11.475+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:18:11.484+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:18:11.480+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:11.657+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:18:11.638+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:18:11.663+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:11.794+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.364 seconds
[2023-10-04T13:18:16.540+0000] {processor.py:157} INFO - Started process (PID=607) to work on /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:16.542+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/etl_dag.py for tasks to queue
[2023-10-04T13:18:16.544+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:18:16.543+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:16.600+0000] {logging_mixin.py:151} INFO - [2023-10-04T13:18:16.595+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/etl_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_dag.py", line 3, in <module>
    from airflow.operators.python import PythonOperator, DummyOperator
ImportError: cannot import name 'DummyOperator' from 'airflow.operators.python' (/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py)
[2023-10-04T13:18:16.602+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_dag.py
[2023-10-04T13:18:16.630+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/etl_dag.py took 0.097 seconds
